docs/GettingStartedDocs/SimulatorGettingStarted.md

```diff
@@ -22,11 +22,11 @@ Ansible is required to install the project prerequisites. You can install it by
 sudo ./scripts/ansible/install-ansible.sh
 ```
 
-The ansible-playbook [scripts/ansible/ansible-include_task.yml](/scripts/ansible/ansible-include_task.yml) was created to be able to run pre-defined ansible tasks on a target host. To make installing the prerequisites less tedious execute the following commands from the root of the source tree:
+To install all the OpenEnclave prerequisites you can execute the `environment-setup.yml` tasks from `linux/openenclave` Ansible role:
```

Comment:
Small nit, but we have to write this as "Open Enclave" (always a space, never "OpenEnclave", with the exception of like variable/folder/programmatic names).